src/lib/components/works_for_you/notification.tsx

```diff
@@ -10,14 +10,15 @@ import colors from "../../../data/colors"
 
 interface Props extends RelayProps {}
 
-class Notification extends React.Component<Props, any> {
+export class Notification extends React.Component<Props, any> {
   render() {
     const notification = this.props.notification
 
     return (
       <View style={styles.container}>
         <View style={styles.header}>
-          <View>
+          { notification.image && <Image source={{uri: notification.image.resized.url}} style={styles.artistAvatar}/> }
```

Comment:
only a suggestion having not seen the designs but we should consider rendering a placeholder image if `notification.image` is null? 

so there's no blank space or layout shifts because of that.